compiler/src/dotty/tools/dotc/util/Collections.scala

```diff
@@ -0,0 +1,39 @@
+/* NSC -- new Scala compiler
```

Comment:
What would `map2` be decorated on? Anyway, I'll just remove those methods in favor of the standard library, it shouldn't make much of a difference.